libantworld/world.cc

```diff
@@ -451,6 +479,12 @@ void World::Texture::bind() const
     glBindTexture(GL_TEXTURE_2D, m_Texture);
 }
 //----------------------------------------------------------------------------
+void World::Texture::unbind() const
+{
+    // Unbind texture
+    glBindTexture(GL_TEXTURE_2D, 0);
```

Comment:
Is the ``0`` a ``nullptr``? If so, could be explicit about that

Reply:
no - OpenGL handles are integers

Reply:
Ahh yeah of course.